src/views/Register.vue

```diff
@@ -74,6 +74,8 @@
 </template>
 
 <script>
+import authService from '../api/authService';
```

Comment:
you can directly import the register method:

`import {register}  from '@/api/authService';`